python/ray/job_config.py

```diff
@@ -50,6 +51,7 @@ def __init__(
         ray_namespace: Optional[str] = None,
         default_actor_lifetime: str = "non_detached",
         _py_driver_sys_path: Optional[List[str]] = None,
+        py_log_config: Optional[Union[dict, str]] = None,
```

Comment:
Let's create a LoggingConfig class

Reply:
```python
class LoggingConfig:
    dict_config: Union[dict, str] = "TEXT"
    log_level: int = INFO
```

Reply:
Is there any reason to create a `LoggingConfig` class? With `LoggingConfig`, user's code will look like:

```python
ray.init(job_config=ray.job_config.JobConfig(py_log_config=ray.xxx.yyy.zzz.LoggingConfig("TEXT")))
```

Reply:
> Is there any reason to create a LoggingConfig class?

Maybe this is because we might add new fields to the LoggingConfig in the future?

Reply:
Updated https://github.com/ray-project/ray/pull/45344/commits/bba3b09ddc60369ba398ab417fd4e3a8e5f459c1